sdk/js/src/solana/wormhole/coder/instruction.ts

```diff
@@ -1,41 +1,88 @@
-import { Idl, InstructionCoder } from "@project-serum/anchor";
-import { ETHEREUM_KEY_LENGTH } from "../../utils";
+import { Idl, Instruction, InstructionCoder } from "@project-serum/anchor";
+import { bs58 } from "@project-serum/anchor/dist/cjs/utils/bytes";
```

Comment:
I didn't catch this the first time I reviewed it. Is this going to cause any issues with esm?

Reply:
Ok, so I just solved this by explicitly naming bs58 in `package.json`. bs58 was already in package-lock.json so I don't think this increases the size of the package.